gemfiles/activerecord_5.1.gemfile.lock

```diff
@@ -57,4 +58,4 @@ DEPENDENCIES
   wwtd
 
 BUNDLED WITH
-   2.0.1
+   2.0.2
```

Comment:
Was `bundler` supposed to be updated here?

Reply:
We don't think it really matters, but we can bump it down if the upstream maintainer wants to it be locked to `2.0.1`